### PR TITLE
feat(Popover): add props to improve accessibility

### PR DESCRIPTION
**Pull request by zamkovskaya**
https://github.com/gravity-ui/uikit/issues/691
https://github.com/gravity-ui/uikit/issues/684

### CODE REVIEW

File: src/components/Popover/README.md [hunk 1]
```diff
@@ -38,6 +38,11 @@ Extends [`QAProps`](../types.ts).
...
+| focusTrap               | `Boolean`          |          |                                          | `false`             | Prevent focus from leaving the popover while open                                                                                                                              |
+| autofocus               | `Boolean`          |          |                                          | `false`             | If true, focus will be transferred to the first element when the popover opens                                                                                                 |
+| restoreFocusRef         | `React.RefObject`  |          |                                          |                     | Focused element when the popover closes open                                                                                                                                   |
```
Comment (kseniya57) [inline src/components/Popover/README.md:45]: There seems to be a typo in the description

**Review by kseniya57 — APPROVED** [review]

**Review by amje — CHANGES_REQUESTED** [review]

File: src/components/Popup/Popup.tsx [hunk 2]
```diff
@@ -139,23 +150,29 @@ export function Popup({
...
-                            />
-                        )}
-                        {children}
-                    </div>
+                    <FocusTrap enabled={focusTrap && open} disableAutoFocus={!autoFocus}>
+                        {/* The event handlers should only be used to capture bubbled events */}
+                        {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
+                        <div
+                            onClick={onClick}
+                            onKeyDown={onKeyDown}
```
Comment (amje) [inline src/components/Popup/Popup.tsx:158]: Didn't notice this at first. Why we're listening keydown?

Comment (zamkovskaya) [reply src/components/Popup/Popup.tsx:158]: The onClick event handler is currently used to execute an action within a popup when it's clicked. It returns true or false, which determines whether to close the popup. This behavior should also work for keyboard and screen reader users. I can't think of a specific use case for onClick at the moment, but in any case, if there's an action involved, it should be accessible to users who don't use a mouse.

Comment (amje) [reply src/components/Popup/Popup.tsx:158]: I think onClick here is by mistake. Popup should not handle clicks or keyboard events, it's utility component.

Comment (zamkovskaya) [reply src/components/Popup/Popup.tsx]: I marked onClick as deprecated and removed onKeyDown

**Pull request closed without merging**